examples/ts/src/transfer.ts

```diff
@@ -0,0 +1,26 @@
+import {createApi} from "./api";
+import {Keyring} from "@polkadot/api";
+import config from "./config";
+
+/**
+ * Example of transferring tokens from Alice to Bob.
+ */
+async function main() {
+    // instantiate the api
+    const api = await createApi()
+    // construct the keyring after the API (crypto has an async init)
+    const keyring = new Keyring({type: 'sr25519'});
+    // add Alice to our keyring with a hard-derivation path (empty phrase, so uses dev)
+    const alice = keyring.addFromUri(config.mnemonic);
+    // receiver address from config
+    const BOB = config.receiver;
```

Comment:
> can we use the same naming convention for `alice` and `BOB`?

yes, miss that. It make sense to rename it (sender/receiver) since it's fetched from configuration which can be anything.